doc/classes/PackedColorArray.xml

```diff
@@ -59,6 +59,15 @@
 				Creates a copy of the array, and returns it.
 			</description>
 		</method>
+		<method name="fill">
+			<return type="void">
+			</return>
+			<argument index="0" name="value" type="Color">
+			</argument>
+			<description>
+				Assigns the given value to all elements in the array. This can typically be used together with [method resize] to create an array with a given size and initialized elements
```

Comment:
Just missing a dot at the end of the sentence for all `Packed*Array` descriptions, otherwise it's all good.

Reply:
Oh yeah, my fault for copy pasting and not double checking. Thanks for the heads up, it's should be fixed now.